elyra/pipeline/resources/palette.json

```diff
@@ -6,6 +6,7 @@
       "image": "",
       "id": "notebooks",
```

Comment:
Unrelated to this PR, but does anyone know why the `id` for `notebooks` plural when all others are singular?  Even the description field is singular. Unless it has far-reaching implications to other things (presumably on the front-end), it seems like this should be updated for the sake of consistency.  I have no idea where this value is used.

Reply:
We definitely check against the `id` in the backend in a few places, but could easily change that. I don't think there are any additional implications for the backend

But yes, not sure if there is a frontend necessity or not. If not, could be good to rope into the changes to #1797